tests/integration/test_structured_logging_json/test.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from helpers.cluster import ClickHouseCluster
+import logging
+import json
+from xml.etree import ElementTree
```

Comment:
Now it seems like we don't need those anymore :)
```suggestion
import json
```

Reply:
code updated